lib/wings.rb

```diff
@@ -55,7 +55,25 @@
 #
 # @see https://wiki.duraspace.org/display/samvera/Hyrax-Valkyrie+Development+Working+Group
 #      for further context regarding the approach
-module Wings; end
+module Wings
+  ##
+  # @api public
+  #
+  # Provides a search builder for new valkyrie types that are indexed as their
+  # corresponding legacy ActiveFedora classes.
+  #
+  # @example
+  #   builder = Wings::WorkSearchBuilder(Monograph)
+  def self.WorkSearchBuilder(work_type) # rubocop:disable Naming/MethodName
```

Comment:
Should we re-enable these cops down around line 75 or so?

Reply:
i only recently learned this, but the end-of-line disable only applies to the one line! super handy

Reply:
oh! that is super nice. TIL. thank you.